app/controllers/sessions_controller.rb

```diff
@@ -1,7 +1,17 @@
-class SessionsController < Clearance::SessionsController
-  protected
+class SessionsController < ApplicationController
+  skip_before_action :verify_authenticity_token, only: :new
```

Comment:
I am not clear on why this line is here. Why would you need to skip this verification?